bidding_auction_services_multi_seller_auctions.md

```diff
@@ -306,6 +306,21 @@ The updated definition for AuctionResult will be as follows:
 _Note: Additional metadata will be sent back to the client for reporting (URL generation and report URL ping) after
 the top-level auction on the browser. The API changes will be published at a later date in a different explainer._
 
+
+#### Ad Tech Specifications
+##### Component Seller Ad Server
+The seller's ad server has to orchestrate the flow to conduct the component auction which includes - 
+1. Sending a SelectAdRequest to SellerFrontEnd service with a B&A ciphertext and an auctionConfig as described in the [API changes][28].
+
+_Note: For a component auction, the auctionConfig must contain a ‘topLevelSeller’ field with the domain of the top level seller which will call runAdAuction on the device._
+
+2. Including the encrypted AuctionResult responses (returned from the SelectAd RPC in step 1) in the response to the browser. 
```

Comment:
Includes ..............SelectAd request .... sent back in the HTTP response body back to the [JavaScript Fetch initiated by the browser with "adAuctionHeaders: true"](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md#step-2-send-auction-blob-to-servers). 

(s/RPC/request since they can send HTTPS too, translated by Envoy)